### ISSUE TITLE
spanner column's default value not supported

### ISSUE THREAD

**Issue by laljikanjareeya** [googleapis/dotnet-spanner-entity-framework#4]
As Google Cloud Spanner Does not support column's default value Below annotation will not work:

```C#
protected override void OnModelCreating(ModelBuilder modelBuilder)
{
........

entity.Property(e => e.CreateDate)
.HasDefaultValue(DateTime.UtcNow);

.......
}
```

**Comment by kt81** [comment]
Cloud Spanner now supports setting default values for columns, correct?
Are there any plans to implement support for this?

**Comment by olavloite** [comment]
> Cloud Spanner now supports setting default values for columns, correct? Are there any plans to implement support for this?

Yes, this was added in #439 and has now been released in version 2.2.0
